### PR TITLE
(MAINT) Backport docs update for CRL/HUP

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

File: documentation/restarting.markdown [hunk 1]
```diff
@@ -35,10 +35,11 @@ There are three ways to trigger your Puppet Server environment to refresh and pi
...
+* Changes to the CA CRL file.  For example, a `puppet cert clean`
 
 ### Changes that require a full Server restart
 
 * Changes to JVM arguments, such as [heap size settings](./tuning_guide.html#jvm-heap-size), that are typically configured in your `/etc/sysconfig/puppetserver` file.
 * Changes to [`bootstrap.cfg`](./configuration.html#service-bootstrapping) to enable or disable Puppet Server's certificate authority (CA) service.
 
-For these types of changes, you must restart the process by using the operating system's service framework, for example, by using the `systemctl` or `service` commands.
\ No newline at end of file
+For these types of changes, you must restart the process by using the operating system's service framework, for example, by using the `systemctl` or `service` commands.
```
Comment (cprice404) [inline documentation/restarting.markdown]: I have no idea why this is showing up as a diff.

Comment (cprice404) [inline documentation/restarting.markdown]: Also don't care much unless you do, @gguillotte .

Comment (gguillotte) [inline documentation/restarting.markdown]: I don't. Your text editor probably added a newline to the end of the file.

Comment (cprice404) [inline documentation/restarting.markdown]: That's what I thought but I spent a few minutes trying to mess with that and it didn't seem to do anything.

Anyway, if you are ok with this as-is, you can merge it and I'll deal with merging up to the other branches.

**Comment by gguillotte** [comment]
👍

**Pull request merged**